REQUIRE

```diff
@@ -1,5 +1,5 @@
-julia 0.5
-Compat 0.17.0
+julia 0.6
+Compat 0.48.0
```

Comment:
0.52 for objectid https://github.com/JuliaLang/Compat.jl/commit/02367e6be37acfc319920ca593c5ae0cb9854ce4

Reply:
Ah, good catch, thanks!